digitalpublicgoods/pharmacovigilance-electronic-reporting-tool.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "Pharmacovigilance Electronic Reporting Tool",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://pv.pharmacyboardkenya.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.google.com/document/d/1LjkPOKrK-4O_PBb4GRfx61I2_0zfesD_5X1y9aw34dQ/edit#heading=h.6nlwssopwwv3"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "No",
+    "nonPIIAccessMechanism": ""
```

Comment:
PvERS II needs to clarify how non-PII data is imported or exported from the platform. Thanks.

Reply:
Fixed in commit fcc426e